src/v2/Apps/Order/Components/PriceOptions.tsx

```diff
@@ -0,0 +1,145 @@
+import { OfferInput } from "v2/Apps/Order/Components/OfferInput"
+import { compact } from "lodash"
+import { createFragmentContainer, graphql } from "react-relay"
+
+import { PriceOptions_artwork } from "v2/__generated__/PriceOptions_artwork.graphql"
+import { appendCurrencySymbol } from "../Utils/currencyUtils"
+
+export interface PriceOptionsProps {
+  setValue: (value: number) => void
+  onFocus: () => void
+  showError?: boolean
+  artwork: PriceOptions_artwork | null | undefined
+}
+
+export const PriceOptions: React.FC<PriceOptionsProps> = ({
+  setValue,
+  onFocus,
+  showError,
+  artwork,
+}) => {
+  const asCurrency = (value: number) =>
+    value?.toLocaleString("en-US", {
+      currency: artwork?.priceCurrency!,
+      minimumFractionDigits: 2,
+      style: "currency",
+    })
+
+  const [customValue, setCustomValue] = useState<number>()
+  useEffect(() => {
+    customValue && setValue(customValue)
+  }, [customValue])
+
+  const [toggle, setToggle] = useState(false)
+
+  const listPrice = artwork?.listPrice
+  const minPriceRange = listPrice?.minPrice?.major
+  const maxPriceRange = listPrice?.maxPrice?.major
+  const midPriceRange = (Number(minPriceRange) + Number(maxPriceRange)) / 2
+
+  const getRangeOptions = () => {
+    const getRangeDetails = [
+      { value: minPriceRange, description: "Low-end of range" },
+      { value: midPriceRange, description: "Midpoint" },
+      { value: maxPriceRange, description: "Top-end of range" },
+    ]
+    return getRangeDetails.map(rangePrice => ({
+      value: rangePrice.value!,
+      description: rangePrice.description,
+    }))
+  }
+
+  const getPercentageOptions = () => {
+    return [0.2, 0.15, 0.1].map(pricePercentage => {
+      if (listPrice?.major) {
+        return {
+          value: listPrice.major * (1 - pricePercentage),
+          description: `${pricePercentage * 100}% below list price`,
+        }
+      }
+      return
+    })
+  }
+
+  const priceOptions = artwork?.isPriceRange
+    ? getRangeOptions()
+    : getPercentageOptions()
+
+  return (
+    <RadioGroup>
+      {compact(priceOptions)
+        .map(({ value, description }) => (
+          <BorderedRadio
+            value={`price-option-${value}`}
+            label={appendCurrencySymbol(
+              asCurrency(value!),
+              artwork?.priceCurrency!
+            )}
+            onSelect={() => {
+              setValue(value!)
+              setToggle(false)
+            }}
+            key={`price-option-${value}`}
+            data-test="price-option"
+          >
+            <Text variant="sm" color="black60">
+              {description}
+            </Text>
+          </BorderedRadio>
+        ))
+        .concat(
+          <BorderedRadio
+            value="custom"
+            label="Different amount"
+            onSelect={() => {
+              customValue && setValue(customValue)
+              setToggle(true)
+            }}
+            data-test="price-option"
```

Comment:
I guess you don't need to specify the `data-test` here to be able to query the radio elements, you can you `screen.getAllByRole("radio")`, also is a good approach avoiding queries by `testId` as much as we can, has the lowest priority from the library
more info on this [link](https://testing-library.com/docs/queries/about/#priority)